BossMod/BossModule/BossModuleConfig.cs

```diff
@@ -1,31 +1,51 @@
 ﻿namespace BossMod;
 
-[ConfigDisplay(Name = "Boss module window settings", Order = 1)]
+[ConfigDisplay(Name = "Boss Modules and Radar", Order = 1)]
```

Comment:
You only moved stuff around, right? Not actually changed variable names? If you did, that would fuck up people's configs.